FrontEnd/app/views/reports/annual_information_statement_report.html

```diff
@@ -4,66 +4,644 @@
         <li><a href="#/reports/all">{{'label.anchor.reports' | translate}}</a></li>
         <li class="active">{{'label.anchor.runreport' | translate}}</li>
     </ul>
-    <api-validate></api-validate>
-	    <div class="span gray-head" style="margin-left:0%;height:30px;">
-	          <span style="margin-left: 10px;font-size:20px;">
-	                <strong>Annual Information Statement</strong>
-	          </span>
+	<div class="card well">
+    	<api-validate></api-validate>
+    	<div data-ng-hide="hideHeader" class="span gray-head" style="margin-left:0%;height:30px;">
+	        <span style="margin-left: 10px;font-size:20px;">
+	        <strong>{{AIS}}</strong>
 	    </div>
-	    <div ng-hide="isCollapsed" class="alert-block form-horizontal">
-	        <br>
-
+	    <div data-ng-hide="hideTable">
+		    <div class="span gray-head" style="height:auto; text-align: center;">
+		    	<span style="font-size:20px;">
+		          <strong>{{SEC}}</strong><br><br>
+		          <strong>{{AIS}}</strong><br>
+		          <strong>SEC FORM 85-18-1 / 2013-IS-ECP</strong><br>
+		          <strong style="font-size:15px;">Issuer of Exempt Commercial Papers-</strong><br>
+		          <strong style="font-size:15px;">filed pursuant to SRC Rule 9.2 and 17.1 (2)(C)-</strong ><br>
+		          <strong style="font-size:15px;">For the year: <input ng-model="AISYear" type="text"style="width:50px"></strong><br>
+		      </span>
+		    </div>
+		    <br>
 	        <div class="form-group info">
-	            <label class="control-label col-sm-2 ng-binding">Office</label>
+		        <div style="margin-left: 2%;font-size:15px;">
+		       		<span><label style="width:20%">Corporate Name: </label><input ng-model="CorpName"type="text" style="width:70%"></span>
```

Comment:
Put the static labels in global-translations/locale-en.json
and use that as the static label
ex.
<li class="active">{{'label.anchor.runreport' | translate}}</li>